portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,35 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id="header">
+            <h1 id="headerText">BIBI HASSAN</h1>
+            <h2>"Feet on the ground, heart on my sleeves, and head in the clouds." </h2>
+            <h2>Click here for a random fact about me:</h2>
+            <button onclick="addRandomGreeting()">Random Fact</button>
+            <div id="greeting-container"></div>
+      </div>
+      <div id=" pictures">
```

Comment:
Remove the extra space inside of pictures. Actually, i don't see this ID being used anywhere. If its not used anywhere, we can just remove it.